love/src/components/Facility/Map/Levels/Level1.jsx

```diff
@@ -0,0 +1,1814 @@
+import React, { Component } from 'react';
+import lodash from 'lodash';
+import PropTypes from 'prop-types';
+import styles from './Level1.module.css';
+import Device from '../Device.jsx';
+import * as d3 from 'd3';
+
+export default class Level1 extends Component {
+  constructor(props) {
+    super(props);
+    this.mapId = lodash.uniqueId('Map-');
+    this.overlayId = lodash.uniqueId('Overlay-');
+    this.deviceId = lodash.uniqueId('Devices-');
+  }
+
+  static propTypes = {};
```

Comment:
Please add PropTypes :pray:. Should be a copy and past for the rest of the Levels.